mylyn.tasks/connectors/gitlab/org.eclipse.mylyn.gitlab.ui/src/org/eclipse/mylyn/gitlab/ui/GitlabTaskEditorSummaryPart.java

```diff
@@ -0,0 +1,90 @@
+package org.eclipse.mylyn.gitlab.ui;
+
+import org.eclipse.jface.layout.GridDataFactory;
+import org.eclipse.mylyn.commons.identity.core.spi.ProfileImage;
+import org.eclipse.mylyn.gitlab.core.GitlabCoreActivator;
+import org.eclipse.mylyn.gitlab.core.GitlabRepositoryConnector;
+import org.eclipse.mylyn.internal.tasks.ui.TasksUiPlugin;
+import org.eclipse.mylyn.internal.tasks.ui.editors.EditorUtil;
+import org.eclipse.mylyn.internal.tasks.ui.editors.TaskEditorExtensions;
+import org.eclipse.mylyn.internal.tasks.ui.editors.TaskEditorSummaryPart;
+import org.eclipse.mylyn.internal.tasks.ui.editors.UserAttributeEditor;
+import org.eclipse.mylyn.tasks.core.AbstractRepositoryConnector;
+import org.eclipse.mylyn.tasks.core.data.TaskAttribute;
+import org.eclipse.mylyn.tasks.ui.AbstractRepositoryConnectorUi;
+import org.eclipse.mylyn.tasks.ui.TasksUi;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.FocusAdapter;
+import org.eclipse.swt.events.FocusEvent;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.ui.forms.widgets.FormToolkit;
+
+import com.google.common.base.Strings;
+
+public class GitlabTaskEditorSummaryPart extends TaskEditorSummaryPart {
```

Comment:
Shouldn't we move it to "internal" package?

Reply:
done with next commit